merkle/coniks/coniks.go

```diff
@@ -57,9 +57,13 @@ func (m *Hasher) HashEmpty(treeID int64, root node.ID) []byte {
 	buf := bytes.NewBuffer(make([]byte, 0, 32))
 	h := m.New()
 	buf.Write(emptyIdentifier)
-	binary.Write(buf, binary.BigEndian, uint64(treeID))
+	if err := binary.Write(buf, binary.BigEndian, uint64(treeID)); err != nil {
```

Comment:
Interesting that the `buf.Write` didn't trigger the linter, that's very clever...

Anyway, given that these `binary.Write` calls in here (and below on L82 etc.) are writing to a `bytes.Buffer`, I think any errors will `panic` rather than return (https://cs.opensource.google/go/go/+/refs/tags/go1.20.3:src/bytes/buffer.go;l=164-165), so you _could_ opt to `_ = ...` these with a comment if you liked?

Reply:
Ta!